lib/DidKeyDriver.js

```diff
@@ -34,7 +19,11 @@ export class DidKeyDriver {
   constructor({verificationSuite = Ed25519VerificationKey2020} = {}) {
     // used by did-io to register drivers
     this.method = 'key';
-    this.verificationSuite = verificationSuite;
+    this.verificationSuite = _createVerificationSuite({
```

Comment:
This helper was meant to be used externally -- by the creator of the `DidKeyDriver` instance. There was already an API called a `verificationSuite` that they can just pass in here if they have one. If they don't have one, they can create one by using `createVerificationSuite`, a new utility method we expose to them. They shouldn't call that if they already have something like `Ed25519VerificationKey2020` or `Ed25519VerificationKey2018` -- they can just pass that in directly without using the helper.